### ISSUE TITLE
Undefined offset error in AbstractState

### ISSUE THREAD

**Issue by johannes85** [xp-framework/scriptlet#9]
In scriptlet/xml/workflow/AbstractState under specific conditions an "Undefined offset error" is raised:

See line 150:
```php
// Load handler values from session
// If active, remove form params, they'll be read from $request now.
$this->handlers[$i]->values= $request->session->getValue($this->handlers[$i]->identifier);
if (!$this->handlers[$i]->requestOverride && $this->handlers[$i]->isActive($request, $context)) {
  unset($this->handlers[$i]->values[HVAL_FORMPARAM]);
}
$node->setAttribute('status', HANDLER_INITIALIZED);
$this->addHandlerToFormresult($this->handlers[$i], $node, $request);
```

If requestOverride is set to false for an handler and it is active, ```$this->handlers[$i]->values[HVAL_FORMPARAM]``` will be unset which causes an error in ```$this->addHandlerToFormresult``` since it tries to access the unset key (``` foreach (array_keys($handler->values[HVAL_FORMPARAM]) as $key) {```).

**Comment by thekid** [comment]
Good catch!

**Comment by thekid** [comment]
:shipit: Released in [8.0.2](https://github.com/xp-framework/scriptlet/releases/tag/v8.0.2)
